### DISCUSSION TITLE
Cross compatibility question. Would Dexcom ONE transmitter work with G6 sensor?

### DISCUSSION THREAD

**Discussion by aleks372** [NightscoutFoundation/xDrip#2580] [Q&A]
Does anyone have experience or maybe knows the technical aspects.

Would the Dexcom ONE transmitter work with the classic G6 sensor?

Just very interested in whether, in a situation where the G6 sensor is available and the transmitter is Dexcom ONE, if they could be made to work together.
P.S. Because in some European markets the One transmitters are many times cheaper than the G6 ones.

**Comment by hidoba** [comment]
@aleks372 Have you figured it out?

**Reply by aleks372** [reply]
They should work fine, as Dex ONE and G6 is almost the same system.
Just use the sensor calibration code and all should be fine.

**Reply by Navid200** [reply]
If using xDrip, you can ignore the conversion.
Just enter the code from the sensor you are using.  Don't enter a different code.
